en/guides/teachers/getting-started/index.md

```diff
@@ -1,74 +1,93 @@
+## What is Dodona?
 
-Op Dodona kunnen lesgevers een eigen cursus aanmaken, opgebouwd uit verschillende oefeningenreeksen. Ze kunnen hiervoor beroep doen op bestaande cursussen en oefeningen, maar kunnen er ook voor kiezen om zelf aan de slag te gaan en eigen oefeningen en lesmateriaal op te stellen. Dankzij de ingebouwde *learning analytics* en datavisualisaties is het bovendien eenvoudig om de voortgang van studenten te meten.
+Dodona is an online exercise platform for learning to code.
+The web application was developed at Ghent University to support practical exercises for large groups of students.
+Dodona therefore primarily focuses on the automatic testing and feedback of submitted solutions to programming exercises.
+Currently, Dodona supports programming languages such as Python, JavaScript, Java, Kotlin, C#, bash, Prolog, Haskell, SQL, and R.
 
-Dodona wordt gratis aangeboden voor educatieve doeleinden.
+On Dodona, teachers can create their own course, built from different exercise series.
+They can use existing courses and exercises or choose to create their own exercises and teaching materials.
+Thanks to the built-in *learning analytics* and data visualizations, it is also easy to measure students' progress.
 
-## Aanmelden
+## Logging In
 
-Aanmelden op Dodona kan via de Smartschool, Microsoft 365 of Google workspace account van je school. Dodona krijgt vervolgens je naam, e-mailadres, en naam van je school doorgestuurd van het gekozen loginsysteem. Op basis hiervan kunnen we je identificeren en maken we automatisch een Dodona-account aan de eerste keer dat je aanmeldt. Je wachtwoord wordt op geen enkel moment naar ons doorgestuurd.
+You can log in to Dodona using your school's Smartschool, Microsoft 365, or Google Workspace account.
+Dodona then receives your name, email address, and school name from the chosen login system.
+Based on this information, we can identify you and automatically create a Dodona account the first time you log in.
+Your password is never sent to us.
 
-::: tip Meerdere inlogsystemen?
-Sommige scholen bieden zowel inloggen via Smartschool als via Microsoft 365 of Google workspace aan. Omdat je Dodona-account gekoppeld is aan het gebruikte inlogsysteem, zal je twee verschillende Dodona-accounts hebben als je bijvoorbeeld eerst via Smartschool aanmeldt en vervolgens via Microsoft 365.
+::: tip Multiple login systems?
+Some schools offer login via both Smartschool and Microsoft 365 or Google Workspace.
+Since your Dodona account is linked to the login system used, you will have two different Dodona accounts if you log in via Smartschool first and then via Microsoft 365.
```

Comment:
```suggestion
Since your Dodona account is linked to the sign-in system used, you will have two different Dodona accounts if you sign in via Google first and then via Microsoft 365 next.
```